client/my-sites/marketing/do-it-for-me/landing.tsx

```diff
@@ -1,82 +1,153 @@
+import consult1 from 'calypso/assets/images/difm/consult-1.png';
+import consult2 from 'calypso/assets/images/difm/consult-2.png';
+import dietician from 'calypso/assets/images/difm/dietician-1.png';
+import musician from 'calypso/assets/images/difm/musician-1.png';
+import photography from 'calypso/assets/images/difm/photography-1.png';
+import phsych from 'calypso/assets/images/difm/phsych-1.png';
+import restaurant from 'calypso/assets/images/difm/restaurant-1.png';
+import treeremoval from 'calypso/assets/images/difm/tree-removal.png';
+import { FullWidthButton } from 'calypso/my-sites/marketplace/components';
 import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
 
 const VerticalsGrid = styled.div`
 	display: grid;
 	grid-template-columns: repeat( 3, 1fr );
-	gap: 2em;
+	gap: 0.75em;
+	text-align: center;
 	@media ( max-width: 960px ) {
 		grid-template-columns: 1fr;
 	}
 `;
 
+const Vertical = styled.h1`
+	margin-bottom: 16px;
+	padding: 16px 20px;
+	margin: 0 auto 10px;
+	padding: 16px;
+`;
+
+const VerticalHeading = styled.h1`
+	font-size: 1.25rem;
+	text-align: center;
+	margin: 1em 0 1em 0;
+	height: 4.25em;
+	border-bottom: 1px solid var( --color-border-subtle );
+	box-sizing: border-box;
+`;
+
+const ImageContainer = styled.div`
+	height: 150px;
+	overflow: hidden;
+	margin: 10px 0px;
+	@media ( min-width: 960px ) {
+		height: auto;
+	}
+`;
+
+const ButtonContainer = styled.div`
+	margin: 0 15px;
+	width: 100%;
+	@media ( min-width: 480px ) {
+		width: 200px;
+	}
+`;
+
+const Paragraph = styled.p`
+	margin: 16px;
+	font-size: 1rem;
+	text-align: justify;
+`;
+
+const Heading = styled.h2`
+	font-size: 1.5rem;
+	margin: 15px 15px 0;
+	padding: 0;
+`;
+
 export default function DoItForMeLandingPage(): JSX.Element {
 	const selectedSiteSlug = useSelector( getSelectedSiteSlug );
+	const translate = useTranslate();
 
 	const onInterestedSelected = async () => {
 		page( `/marketing/do-it-for-me/site-info/${ selectedSiteSlug }` );
 	};
 
 	return (
 		<Card>
-			<FormattedHeader
-				brandFont
-				headerText="Your Site Built By Wordpress"
-				subHeaderText="You want the website of your dreams. Our experts can create it for you."
-				align="left"
-			/>
-			<img
-				src="https://wpcom.files.wordpress.com/2020/10/built-by-hero-image.png"
-				width="100%"
-				alt="WordPress logo"
-			/>
+			<Heading className="do-it-for-me__header formatted-header__title wp-brand-font">
+				{ translate( 'Your Site Built By WordPress.com' ) }
+			</Heading>
 
-			<CardHeading tagName="h5" size={ 24 }>
-				Our website building plans are perfect for:
-			</CardHeading>
+			<Paragraph>
+				{ translate(
+					'Need a professionally designed website for your business fast? Choose from our custom designed templates below, add your content and media, and in just 1-2 days your website will be ready to launch, for just US$500. The one-time fee includes a 5-page website, a 30-minute Quick Start Zoom orientation about your site when it’s complete.'
```

Comment:
Some changes suggested by Grammarly:
```suggestion
					'Need a professionally designed website for your business fast? Choose from our custom-designed templates below, add your content and media, and in just 1-2 days, your website will be ready to launch for just US$500. The one-time fee includes a 5-page website and a 30-minute Quick Start Zoom orientation about your site when it’s complete.'
```

Reply:
`US$500`: should we display the actual price in the user's currency? Example here:
https://github.com/Automattic/wp-calypso/blob/033cf1befbddeac547bacd50abc828e42a225fc4/client/my-sites/marketing/ultimate-traffic-guide/index.js#L76
Since this is the pilot, please don't consider this blocking. :)

Reply:
Thanks fixing these 👍